api/src/api/notes.js

```diff
@@ -14,7 +14,9 @@ const {
 } = require('../middleware/notes');
 
 const memberFromId = async (ids) => {
-  const memberPromises = ids.map((memberId) => Member.findById(memberId));
+  const memberPromises = ids
```

Comment:
Add a comment about what this operation is doing, specifically curious about the filter and had to look that up.